coalib/parsing/filters/TagSectionFilter.py

```diff
@@ -0,0 +1,21 @@
+def tag_section_filter(section, args):
+    """
+    Filters the sections by ``tags``. ``cli`` is always accepted.
+
+    :param section: Section object.
+    :param args:    Set of tags on which ``section`` is to be filtered.
+    :return:        ``True`` if this section matches the criteria inside args,
+                    ``False`` otherwise.
+    """
+    if section.name == 'cli':
```

Comment:
I agree with @shreyans800755 that this doesnt belong in the filter.
https://github.com/ksdme/coala/commit/f8fc69c533333d73f891115edd4be14b1d3da458#r29964403

The `cli` section should be kept hidden inside `coalib/settings`, as a special case implementation detail.  It annoys me that we have any UI messages that mention this section -- the user cant see it, so they shouldnt hear about it.

And likewise developers in other parts of coalib shouldnt need to deal with that section.